api/actions/actionsRouter.js

```diff
@@ -1,8 +1,116 @@
 const express = require('express');
-const { validateSubjectBody } = require('./actionsMiddleware');
-const Actions = require('./actionsModel');
 const router = express.Router();
+const Actions = require('./actionsModel');
+const { validateSubjectBody } = require('./actionsMiddleware');
+/**
+ * @swagger
+ * components:
+ *  schemas:
+ *    Actions:
+ *      type: object
+ *      required:
+ *        - submitted_by
+ *        - subject_id
+ *        - issue
+ *      properties:
+ *        action_ticket_id:
+ *          type: integer
+ *          description: Unique primary key referencing an action ticket
+ *        submitted_by:
+ *          type: integer
+ *          description: Unique primary key referencing a profile's auto-assigned ID
+ *        subject_id:
+ *          type: integer
+ *          description: Unique primary key referencing a profile's auto-assigned ID
```

Comment:
It's not clear what this property is referencing, exactly, just from reading this. `submitted_by` is obviously referencing the person who wrote the ticket, but `subject_id` is actually referencing someone who is the subject of the ticket, itself (for example, a mentee that a mentor is teaching, or a mentor that a mentee is working with). This property should not have an identical description to the one above it, as they are different data points, even though they both reference profile IDs.